src/Main.hs

```diff
@@ -83,6 +94,31 @@ noEdges = AllEdges { normalEdges    = IMap.empty
                    , sourceEdges    = IMap.empty
                    }
 
```

Comment:
This is rather mysterious, could you name it something more meaningful and maybe add a comment as to what we are doing here.